src/bin/exrmultipart/exrmultipart.cpp

```diff
@@ -419,6 +419,8 @@ combine (
 
                 partnums.push_back (j);
             }
+            
+            delete infile;
```

Comment:
I don't think this is correct. `infile` gets added to the `fordelete` vector above, and deleted below, so it shouldn't be leaking.

Is it possible that your other fix resolved the leak itself?

`test_exrmultipart.py` is failing in the CI checks, this is presumably why.

Thanks!

Reply:
Yes, I'd missed the `fordelete` thing. The actual leak seems to be in the missing `delete` when erasing an attribute.